packages/@uppy/dashboard/src/components/FileItem/FileProgress/index.scss

```diff
@@ -0,0 +1,159 @@
+.uppy-DashboardItem-progress {
+  display: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: $zIndex-3;
+  color: $white;
+  text-align: center;
+  width: 120px;
+  transition: all .35 ease;
+
+  > .uppy-DashboardItem-progressIndicator {
+    display: inline-block;
+    width: 38px;
+    height: 38px;
+    opacity: 0.9;
+    cursor: pointer;
+    &:focus{
+      outline: none;
+      svg.UppyIcon-progressCircle circle.bg,
+      svg.retry{
+        fill: lighten($blue, 20%);
+      }
+    }
+  }
+}
+
+svg.UppyIcon-progressCircle {
+  width: 100%;
+  height: 100%;
+  circle.bg {
+    stroke: rgba($white, 0.4);
+    opacity: 0;
+  }
+  circle.progress {
+    stroke: $white;
+    transition: stroke-dashoffset .5s ease-out;
+    opacity: 0;
+  }
+  polygon.play {
+    stroke: $white;
+    fill: $white;
+    opacity: 0;
+    transition: all 0.2s;
+    display: none;
+  }
+  polygon.cancel {
+    fill: $white;
+    opacity: 0;
+    transition: all 0.2s;
+  }
+  g.pause {
+    stroke: $white;
+    fill: $white;
+    opacity: 0;
+    transition: all 0.2s;
+    display: none;
+  }
+  polygon.check {
```

Comment:
I personally think `polygon`, `circle` are implementation detail and unnecessarily specific. Tomorrow we might change the SVG icon, and `g` group is not needed.

Reply:
@arturi, isn't it more clear to you when you read `circle.progress {}` rather than `.progress {}`?
When I look at `polygon.check {}` I know immediately, that it's some part of an svg we're talking about here.

> Tomorrow we might change the SVG icon, and g group is not needed.

Exactly, and we'll delete everything easily and safely, just by deleting everything under the `> svg {}` namespace.
That's about nesting though, rather than about including tag names, tag names are strictly for readability. Do you think we're better off without them?